src/Language/Haskell/Liquid/Constraint/Monad.hs

```diff
@@ -100,13 +101,14 @@ addLocA !xo !l !t
 
 addHole :: Var -> SpecType -> CGEnv -> CG () 
 addHole x t γ = do 
```

Comment:
@pbougou can you flag this code? 
e.g., `addHoles` should do all these if `typedHoles (getConfig γ)`, otherwise do nothing